lib/__tests__/test.js

```diff
@@ -0,0 +1,4 @@
+jest.autoMockOff();
```

Comment:
You should put `"use strict";` as the first line of all code files. Add the ESLint rule `strict: ["error", "global"]` to enforce this.

Reply:
Can we just use .mjs as much as possible?

Reply:
Not in Node.js, no.